10up/simple-podcasting
# Type of show setting

### Description of the Change

Adds "Type of show" setting and feed tag to meet Apple Podcaster Guidelines: https://help.apple.com/itc/podcasts_connect/#/itcb54353390

Closes #148

### Verification Process

1. Add new podcast or edit existing
2. See "Type of show" setting dropdown
3. Select either "Episodic" or "Serial" and save the podcast
4. Check podcast RSS — expected to see `<itunes:type>` in the channel section.
5. If "N/A" is selected — the `<itunes:type>` tag should not appear in the channel section

### Checklist:






### Changelog Entry

Added - Type of show setting for the podcast

### Credits


Props @cadic @faisal-alvi
